docs/docs/cmd/spo/file/file-roleassignment-remove.mdx

```diff
@@ -57,6 +63,17 @@ Remove a role assignment by group name from a file by id.
 m365 spo file roleassignment remove --webUrl "https://contoso.sharepoint.com/sites/contoso-sales" --fileId "b2307a39-e878-458b-bc90-03bc578531d6" --groupName "saleGroup"
 ```
 
+Remove a role assignment by entra group id from a file by id without prompt
+
+```sh
+m365 spo file roleassignment remove --webUrl "https://contoso.sharepoint.com/sites/contoso-sales" --fileId "b2307a39-e878-458b-bc90-03bc578531d6" --entraGroupId 3c97e01e-978d-417b-a196-b6a3e37fa873 --force
+
```

Comment:
seems like we are missing the closing of code block

```suggestion
```sh
m365 spo file roleassignment remove --webUrl "https://contoso.sharepoint.com/sites/contoso-sales" --fileId "b2307a39-e878-458b-bc90-03bc578531d6" --entraGroupId 3c97e01e-978d-417b-a196-b6a3e37fa873 --force
```

```